_config.yml

```diff
@@ -43,10 +43,18 @@ defaults:
       type: events
     values:
       layout: event
+  - scope:
+      path: ""
+      type: resources
+    values:
+      layout: resource
 
 collections:
   events:
     output: true
     permalink: /:year-:month-:day.html
   posts:
     permalink: /:year-:month-:day-:title.html
+  resources:
+    output: true
+    permalink: /:title.html
```

Comment:
Well...
Right now it is /resources/tonejs/index.html which looks like /resources/tonejs/ in the browser but for it to be /resources/tonejs will require url rewriting which IMHO is way more work than necessary. Let me know if you know a better way.